frontend/common/SliderServerClient.js

```diff
@@ -53,30 +71,33 @@ export const slider_server_actions = ({ setStatePromise, launch_params, actions,
 
     notebookfile_hash.then((x) => console.log("Notebook file hash:", x))
 
-    const bond_connections = notebookfile_hash
-        .then((hash) => fetch(trailingslash(launch_params.slider_server_url) + "bondconnections/" + hash))
-        .then(assert_response_ok)
-        .then((r) => r.arrayBuffer())
-        .then((b) => unpack(new Uint8Array(b)))
-
-    bond_connections.then((x) => {
-        console.log("Bond connections:", x)
-        setStatePromise(
-            immer((state) => {
-                state.slider_server.connecting = false
-                state.slider_server.interactive = Object.keys(x).length > 0
-            })
-        )
-    })
+    let bond_connections = null
 
     const mybonds = {}
     const bonds_to_set = {
         current: new Set(),
     }
     const request_bond_response = debounced_promises(async () => {
+        if (!bond_connections) {
```

Comment:
This part will continue to fetch bondconnections until they are received successfully